### ISSUE TITLE
Installation problem with watchdog module

### ISSUE THREAD

**Issue by sundayoyeniyi** [icing/mod_md#154]
Hi

I encountered the error below when I execute make.  My environment are:

- dockerized centos 8
- httpd 2.4.41
- mod 2.2.0
- apr 1.7.0
- apr-util 1.6.1

```
[root@1d4cd16adc55 mod_md-2.2.0]# make
Making all in src
make[1]: Entering directory '/root/mod_md-2.2.0/src'
  CC       mod_md_la-mod_md_drive.lo
mod_md_drive.c: In function 'run_watchdog':
mod_md_drive.c:155:14: error: 'result' may be used uninitialized in this function [-Werror=maybe-uninitialized]
         rv = md_job_save(job, result, ptemp);
              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
mod_md_drive.c:74:18: note: 'result' was declared here
     md_result_t *result;
                  ^~~~~~
cc1: all warnings being treated as errors
make[1]: *** [Makefile:901: mod_md_la-mod_md_drive.lo] Error 1
make[1]: Leaving directory '/root/mod_md-2.2.0/src'
make: *** [Makefile:497: all-recursive] Error 1
```

httpd status is ok.

Any idea what I may be doing wrong?

SO

**Comment by icing** [comment]
If compilers could only agree on what to warn about and what not! ;-)
The analysis is half-correct. `result` may be uninitialized, but then also `job->dirty` is never true. But I just added explicit initialization there, does not hurt.

Thanks!

**Comment by icing** [comment]
Fixed in v2.2.2.
